### PR TITLE
Posts by user location

### PR DESCRIPTION
**Pull request by drewwilliams5280**

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -5,6 +5,12 @@
   add_filter '/channels'
   add_filter '/jobs'
   add_filter '/mailers'
+  add_filter '/graphql/types/base_enum.rb'
```
Comment (ggerdsen) [inline spec/rails_helper.rb:8]: Thanks for adding these.

**Review by ggerdsen — COMMENTED** [review]
Looks good to me.

File: app/models/post.rb [hunk 2]
```diff
@@ -18,4 +18,11 @@ class Post < ApplicationRecord
   def increase_ring
     update_column(:ring_min_max, Post.ring_min_maxes[ring_min_max] + 1)
   end
+
+  def self.user_can_increase(user_latitude, user_longitude)
```
Comment (Alex-Desjardins) [inline app/models/post.rb:22]: Great work here

**Review by Alex-Desjardins — APPROVED** [review]
Good work on this!

**Pull request merged**